crates/swc_ecma_transforms_typescript/tests/fixture/issue-7104/case1/input.ts

```diff
@@ -0,0 +1,5 @@
+const foo = <T extends {}>() => {
```

Comment:
Any better place for this test case?

Reply:
Please move this to https://github.com/swc-project/swc/tree/bb80c659b3619c4dd98f34efdfc1094b8d46d114/crates/swc_ecma_parser/tests/typescript